librosa/core/time_frequency.py

```diff
@@ -900,6 +902,70 @@ def octs_to_hz(octs, tuning=0.0, bins_per_octave=12):
     return (float(A440) / 16)*(2.0**np.asanyarray(octs))
 
 
+def A4_to_tuning(A, bins_per_octave=12):
+    """Convert reference frequencies to cents (bin fractions).
```

Comment:
Docstring could be more informative here about what reference frequency means, and how this function should be used.

Maybe something more like:
```
Convert a reference pitch frequency (e.g., `A4=435`) to a tuning estimation, in fractions of a bin per octave.

This is useful for determining the tuning deviation relative to A440 of a given frequency, assuming equal temperament.  By default, 12 bins per octave are used.
```

Note the output wouldn't really be cents, but 1/100ths of a cent.